### PR TITLE
Update to Radar Android SDK 3.2.2

### PR DESCRIPTION
**Pull request by phil-brown**
And add getMatrix API call with Android support

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -134,7 +134,32 @@ class _MyAppState extends State<MyApp> {
...
+          //         'longitude': -73.78779,
+          //       },
+          //       {
+          //         'latitude': 35.99801,
+          //         'longitude': -78.94294,
+          //       },
+          //     ], mode: 'car', units: 'imperial');
+          //   },
+          //   child: Text('getMatrix()'),
+          // )
```
Comment (jrtibbetts) [inline example/lib/main.dart:162]: Does this commented-out block need to be here?

File: ios/Classes/RadarFlutterPlugin.m [hunk 2]
```diff
@@ -435,6 +437,55 @@ - (void)cancelTrip:(FlutterMethodCall *)call withResult:(FlutterResult)result {
...
+        [destinationArray addObject: destination];
+    }
+    RadarRouteMode mode = RadarRouteModeCar;
+    if ([modeStr isEqualToString:@"FOOT"] || [modeStr isEqualToString:@"foot"]) {
+        mode = RadarRouteModeFoot;
+    } else if ([modeStr isEqualToString:@"BIKE"] || [modeStr isEqualToString:@"bike"]) {
+        mode = RadarRouteModeBike;
+    } else if ([modeStr isEqualToString:@"CAR"] || [modeStr isEqualToString:@"car"]) {
+        mode = RadarRouteModeCar;
+    }
```
Comment (jrtibbetts) [inline ios/Classes/RadarFlutterPlugin.m:479]: May as well convert the `modeStr` to lower- or uppercase when it's assigned, and that will save you from having to do double checks for each value.

File: example/lib/main.dart (hunk 1 above)
Comment (phil-brown) [reply example/lib/main.dart:162]: @jrtibbetts this is here to test the getMatrix function. I left it in place for our own internal testing. This PR was opened so long ago I don't remember exactly why I left it commented - I believe it worked sometimes, or required some prerequisites to work. It's nice-to-have for future work.

File: ios/Classes/RadarFlutterPlugin.m (hunk 2 above)
Comment (phil-brown) [reply ios/Classes/RadarFlutterPlugin.m:479]: @jrtibbetts this is just copied from a different function in this same class - probably would be worth it to follow up with a separate refactor PR to clean all this up

**Pull request closed without merging**